scrapy/downloadermiddlewares/__init__.py

```diff
@@ -0,0 +1,15 @@
+class DownloaderMiddleware(object):
+    """Base class for downloader middleware.
+    """
+    def process_request(self, request, spider):
```

Comment:
I wouldn't mind `DownloaderMiddleware`. `BaseSpider` for example was deprecated some time ago.